src/routes/sales/add_sell.tsx

```
@@ -0,0 +1,123 @@
+          <div style={styles.categories}>
+            <button style={styles.categoryButton}>全部</button>
+            <button style={styles.categoryButton}>五金</button>
+            <button style={styles.categoryButton}>飲料</button>
+            <button style={styles.categoryButton}>贈品</button>
+          </div>
+          <div className='w-full py-6 px-3'>
+            <h1 className='text-white font-bold text-2xl'>拍賣</h1>
+            <div className='grid grid-cols-2 gap-1'>
+              { sales.map((p) => ( 
+                  <div className='h-80 p-10 bg-stone-500 hover:bg-stone-600 active:bg-stone-700 focus:outline-none focus:ring focus:ring-violet-300 scale-90 hover:scale-100 relative'>
+                    <h1 className='text-slate-50 font-semibold text-xl absolute left-1 bottom-7'>{p.product}</h1>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)